e2e-tests/tests/add-entity-page.spec.ts

```diff
@@ -20,6 +20,7 @@ studioTest("can add an entity page", async ({ page, studioPage }) => {
   await expect(pageInTree).toHaveCount(1);
   await expect(page).toHaveScreenshot();
   await studioPage.saveButton.click();
+  await page.waitForTimeout(500);
```

Comment:
Added this because the test was failing on my personal computer because the local test generation was taking a while, LMK if it should be removed!